components/Main/Exercises/Exercises.js

```diff
@@ -0,0 +1,52 @@
+import { dataBiceps } from "../../../lib/data";
+import { StyledExercises } from "./StyledExercises";
+import React from "react";
+import Description from "./Description/Description";
+import { useState } from "react";
+import Image from "next/image";
+
+export default function Exercises() {
+  const [show, setShow] = useState(false);
```

Comment:
You can avoid using this state. What could help you is SET (https://levelup.gitconnected.com/how-to-find-unique-values-by-property-in-an-array-of-objects-in-javascript-50ca23db8ccc) to create an array of ID's.
This will help you especially if you want to toggle more than one Card.
For now it work like you did it, but you should tackle this in a future user story :)